Verification_Engine/Compute/TryGetValueFromSource.cs

```diff
@@ -128,6 +128,9 @@ public static Output<bool, object> TryGetValueFromSource(this object obj, Proper
         // with the last segment of the source path (segments = separated by dots).
         private static Output<bool, object> TryGetValueFromSource(this object obj, string sourceName)
         {
+            if (obj is Output<bool, object> objAsOutput)
+                obj = objAsOutput.Item2;
```

Comment:
And if `objAsOutput.Item1 == false`, shouldn't the call be terminated immediately?